docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -1257,48 +1257,36 @@ effect size will then be used to determine the simulant's LBWSG risk category.
 Baseline Calibration -- Hemoglobin and Birth Weight
 '''''''''''''''''''''''''''''''''''''''''''''''''''
 
-- **C_forticant_base** = coverage of forticant in the population at baseline
+A simulant's individual hemoglobin/birthweight value *in the baseline scenario* should be assigned as follows:
 
-- **P_exp_base**  = (1 - C_forticant_base), prevalence of exposure to unfortified foods at baseline
-
-- **π_1** = mean outcome value among those exposed to unfortified foods
-
-- **π_0** = mean outcome value among those unexposed to unfortified foods (covered by fortification)
-
-- **π_GBD** = mean outcome value in overall GBD population for age, sex, location, year
-
-- **effect_size** = (π_0 - π_1), effect size for relevant outcome, as documented in the above sections, dependent on *age*, but **not** *time* (because it is assumed all simulants have been exposed for sufficient duration in the baseline scneario)
-
-- **∆_π_0** = (π_0 - π_GBD), the difference between the mean outcome value in the population unexposed to unfortified foods (covered by fortification) and the overall age-, sex-, and location-specific GBD population
-
-- **∆_π_1** = (π_1 - π_GBD), the difference between the mean outcome value in the population exposed to unfortified foods (not covered by fortification) and the overall age-, sex-, and location-specific GBD population
-
-Equations:
-
-  π_GBD = π_1 * P_exp_base + π_0 * (1 - P_exp_base)
-
-  π_0 - π_1 = effect_size
-
-  ∆_π_0 = π_0 - π_GBD
-
-  ∆_π_1 = π_1 - π_GBD
-
-Step 1: Solve for ∆_π_0 and ∆_π_1 using the equations above
+.. math::
 
-Step 2: Apply ∆_π_0 as the additive effect size for simulants covered by
-baseline coverage and ∆_π_1 as the additive effect size for simulants not
-covered by baseline coverage (apply in the same fashion as described in the
-sections above; notably, this is only done in the iron_responsive_i = 1
-population for the hemoglobin outcome).
+  π_i =	\hat{π}_{GBD(i)} - αp + αx_i
```

Comment:
The notation GBD(i) makes it seem like GBD is a function of i, which is weird.  If you're following James's notation, it should be `\hat{π}_{GBD}(i)`, though James's notation is also a bit weird.  The hats should be unnecessary.  Why not:

`Y_i = Y_GBD - \alpha p + \alpha x_i`,

where

Y_i = individual hemoglobin level or birthweight
Y_GBD = hemoglobin level or birthweight drawn from the distribution in GBD

and \alpha, p, and x_i are defined as below.

Reply:
I think that the hat notation by convention in statistics denotes a sample of a given population parameter, so the idea there is that it is a given draw GBD distribution but I can definitely clarify that. 

Then \hat{π}_{GBD(i)} would be the value for an individual simulant taken from a given draw of the GBD distribution.

So given that I think my preference is keeping the hat notation, but let me know if you still prefer the Y. I can clarify the existing notation to match what I put in this comment and also change \hat{π}_{GBD(i)} to \hat{π}_{GBD}(i)

Reply:
Or I guess to be consistent for our notation for random variables, we should either use X,Y (both capital) or x,y (both lower case).

Reply:
Haha, probability vs. statistics. Wait, I thought a hat denoted an _estimate_ of a population parameter from data.  E.g. if pi_{GBD} is the mean hemoglobin in the population, then \hat{pi}_{GBD} would be an estimate of the population mean, e.g. by measuring Hb levels in 100 people and then computing the average.

https://en.wikipedia.org/wiki/Hat_operator#Estimated_value
https://en.wikipedia.org/wiki/Notation_in_probability_and_statistics#Statistics

But that's not what we're doing here - we're sampling from a distribution with mean pi_{GBD} (well actually, this would be \hat{pi}_{GBD}, because GBD's value is an estimate of the true population parameter), and we don't expect the sampled value to be near the mean, but rather to vary according to the ensemble distribution.

Reply:
Yes! I've generally heard /hat{p} referred to as the sample proportion and p as the population or parameter proportion (generally because you take a *sample* of the population to make an estimate for that population parameter in epi). So I am mixing my "sample" vocab, but I think the concept is similar in that the sample/estimate is a value that is close to the "true value."

Overall I think either way is fine as long as we get the right equation documented. Are you planning to make an updated commit with the latex equations instead of the screenshot or should I plan to make some clarifying edits and keep as is?

Reply:
Why don't you go ahead and keep James's notation for now (just move the (i) out of the subscript) and keep the image, and I'll make a new PR to edit things after this is merged into master.

The main thing I think needs to be fixed before merging is to remove the definition of `\hat{π}_{GBD}` from the table below, and replace it with the definition of `\hat{π}_{GBD}(i)` (see my comment below).

Reply:
should we have a separate repo page to document our standard notations so we are consistent and clear moving forward how to represent things?

Reply:
Ok, it is updated now : )

Reply:
@alibow I don't see the updates.

With regard to notation, from an epidemiology perspective, the hemoglobin level or birthweight (what I called Y_i) would be an exposure or outcome, depending on context. So is the indicator x_i for whether someone gets fortification.  @alibow and @Ninicorn, is there any standard epi convention for naming variables?

I think I've see capital letters used, which matches the convention in probability for random variables, which is how you mathematically model these things, so that's why I chose "Y" above.  In Python the naming convention is to use capital letters for constants or class names, and lower case letters for variables, so that's probably why James uses lower case letters like x_i.  But for this documentation we're aiming for readability of text, not code, so we should choose variable naming conventions that make sense to us.

Reply:
Oops, sorry! My push didn't go through. It is updated now.

Reply:
And yeah, I'd say it varies somewhat but generally is expressed as
Y_i = B0 + B1 * x1_i + B2 * x2_i + ....

where 
Y = outcome
B0 = intercept
B1 = slope for exposure
X1 = exposure
B2 = slope for first covariate
X2 = first covariate

^ This is generally for regression modelling, though.
In more conceptual contexts (like causal diagrams) it's common to see E for exposure and O for outcome and other intuitive letters for additional variables.

It really doesn't matter to me what variable notation we use as long as they are always clearly defined :)